packages/vue/src/components/search/SearchBox.jsx

```diff
@@ -198,12 +199,29 @@ const SearchBox = defineComponent({
 				});
 			}
 
-			suggestionsArray = suggestionsArray.map((s) => {
-				if (s.sectionId) {
-					return s;
-				}
-				return { ...s, sectionId: s._suggestion_type };
-			});
+			suggestionsArray
+			= suggestionsArray
+					.map((s) => {
+						if (s.sectionId) {
+							return s;
+						}
+						return { ...s, sectionId: s._suggestion_type };
+					})
+					.map((suggestion) => {
+						if (suggestion._suggestion_type === 'document') {
+							// Document suggestions don't have a meaningful label and value
+							const newSuggestion = { ...suggestion };
+							newSuggestion.label = 'For recent document suggestion, please implement a renderItem method to display label.';
```

Comment:
suggested typo change -> from `or recent document su...` to `or recent documents su`

Reply:
@siddharthlatest